web/src/components/MapView.tsx

```diff
@@ -115,7 +118,7 @@ const MapView = forwardRef((props: MapViewProps, ref: React.Ref<MapRef>): ReactE
   )
 
   const onDeselect = () => {
-    selectFeature(null)
+    navigate(getParentPath(window.location.pathname))
```

Comment:
```suggestion
    navigate('.')
```
see https://reactrouter.com/en/main/components/link#link

Reply:
fixed in last commit